website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,392 @@ To-do:
 - use the reference doc structure for this article / split into separate articles
 --->
 
+## Iceberg table format <Lifecycle status="beta"/>
+
+Snowflake Iceberg tables support for the dbt-snowflake adapter is available for three out-of-the-box materializations:
+
+- [Table](/docs/build/materializations#table)
+- [Incremental](/docs/build/materializations#incremental)
+- [Dynamic](#dynamic-tables) 
+
+For now, Iceberg tables require a [behavior flag](/reference/global-configs/behavior-changes) due to performance impact related to using Iceberg tables. Snowflake does not support `is_iceberg` on the  `Show Objects` queries, which dbt depends on for metadata.
+
+To use Iceberg, set the `enable_iceberg_materializations` flag to `True` in your dbt_project.yml:
+
+<File name='dbt_project.yml'>
+
+```yaml
+
+flags:
+  enable_iceberg_materializations: True
+
+```
+
+</File>
+
+If you don't set this behavior flag, you will get an error and dbt will not materialize the table in the Iceberg format.
+
+The following configurations are supported: 
+
+| Field | Type | Required | Description | Sample input |
+|-------|------|----------|-------------|-------|
+|Table Format | String | Yes | Configures the objects table format. | `iceberg` |
+|External volume | String | Yes | Specifies the identifier (name) of the external volume where the Iceberg table stores its metadata files and data in Parquet format. If you don’t specify this parameter, the Iceberg table defaults to the external volume for the schema, database, or account. The schema takes precedence over the database, and the database takes precedence over the account.| `my_s3_bucket` |
+|Base location Subpath |String | No | Defines the directory path for Snowflake to write the table data and metadata files. If you change this after creating the table, it will write a new table but leave the existing one. If not specified, dbt-snowflake will default to one based on the model's current schema. | `jaffle_marketing_folder` |
```

Comment:
imho, this table has too many words that are better described elsewhere. To simplify, I suggest:
1. linking to Snowflake's docs page rather than restate their descriptions
2. adding a `Note` column to improve readability and reference elsewhere

### current version

![image](https://github.com/user-attachments/assets/acbe3b99-b772-4939-b98b-c1acb39db8e9)

### render of table with my updates

![image](https://github.com/user-attachments/assets/f325fba1-4955-4196-9446-3456f62cba09)


```suggestion
The following configurations are supported.
For more information, see the Snowflake reference for [`CREATE ICEBERG TABLE` (Snowflake as the catalog)](https://docs.snowflake.com/en/sql-reference/sql/create-iceberg-table-snowflake)

| Field                 | Type   | Required | Description                                                                                                                | Sample input              | Note                                                                                                                                                                                                                                                         |
| --------------------- | ------ | -------- | -------------------------------------------------------------------------------------------------------------------------- | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Table Format          | String | Yes      | Configures the objects table format.                                                                                       | `iceberg`                 | `iceberg` is the only accepted value                                                                                                                                                                                                                          |
| External volume       | String | Yes(*)   | Specifies the identifier (name) of the external volume where Snowflake writes the Iceberg table's metadata and data files. | `my_s3_bucket`            | *You need not specify this if the account, database, or schema already has an associated external volume. [More info](https://docs.snowflake.com/en/sql-reference/sql/create-iceberg-table-snowflake#:~:text=Snowflake%20Table%20Structures.-,external_volume) |
| Base location Subpath | String | No       | An optional suffix to add to the `base_location` path that dbt automatically specifies                                     | `jaffle_marketing_folder` | The recommendation is that you do not specify this. Modifying this parameter results in a new Iceberg table. More info, see [Base Location](#base-location).                                                                                                  |
```